docs/feature_rgb_matrix.md

```diff
@@ -69,6 +69,8 @@ All RGB keycodes are currently shared with the RGBLIGHT system:
 	* `RGB_SAD` - decrease saturation
 	* `RGB_VAI` - increase value
 	* `RGB_VAD` - decrease value
+	* `RGB_SPI` - increase speed effect (no EEPROM support)
+	* `RGB_SPD` - decrease speed effect (no EEPROM support)
```

Comment:
> Speed is not saved in EEPROM yet as per Jack's request.

It appears to be written to EEPROM, isn't it?  https://github.com/qmk/qmk_firmware/pull/2922/files#diff-12394acd867cb721035b8c6151082ba0f9c8c0dfa4115d613b73ade2e41266abR353-R358

Or am I missing something?  (at least I could not find the request, which might not have been with this PR's review)

(I came across this and tried to find when it was changed to use the EEPROM, but it appears to be the case from the beginning already)

Reply:
It appears to have been added in https://github.com/qmk/qmk_firmware/pull/5965 (https://github.com/qmk/qmk_firmware/commit/e2dfb787da2a2ba88e0e074b396a2b988e10eccf); https://github.com/qmk/qmk_firmware/commit/e2dfb787da2a2ba88e0e074b396a2b988e10eccf#diff-46fd16bc8405f0fd698fb8c929877d15aefda76b0ebb46af98a6c4539b8ac8c3R47

Reply:
indeed this is from 2018:)